internal/v1/handler.go

```diff
@@ -1223,3 +1226,27 @@ func (h *Handlers) GetOscapProfiles(ctx echo.Context, distribution Distributions
 	}
 	return ctx.JSON(http.StatusOK, profiles)
 }
+
+func (h *Handlers) GetOscapCustomizations(ctx echo.Context, distribution Distributions, profile DistributionProfileItem) error {
+	globalDist, err := OscapDistributionGeneraization(distribution)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest, err)
+	}
+	//Load the json file with the customizations
+	jsonFile, err := os.Open(path.Join(h.server.OscapDir, filepath.Base(globalDist), filepath.Base(string(profile)), "customizations.json"))
```

Comment:
Maybe we could delegate the file reading and parsing to the `oscap` package? Kind of like we do with `distributions`,
this would tidy this up a bit